docs/SetupAzureDatabase.md

```diff
@@ -51,8 +51,126 @@ Autoscale provisioned throughput in Azure Cosmos DB allows you to scale the thro
 
+### Backup and Restore
+
+Azure Cosmos DB automatically takes backups of your data at regular intervals. The automatic backups are taken without affecting the performance or availability of the database operations. All the backups are stored separately in a storage service. 
+
+More information can be found in [Backup and Restore in Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/online-backup-and-restore) documentation.
+
 ### Monitor Cosmos DB
 
 By default, monitoring is enabled on Cosmos DB.
 
 More information can be found in [Monitor Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/monitor-cosmos-db) documentation.
+
+## MySQL
+
+In this section, you will set up an Azure Database for MySQL servers for Scalar DL.
+
+### Requirements
+
+* You must create an Azure Database for MySQL servers with the plan `Single server`.
+
+### Steps
+
+* Create an Azure Database for MySQL servers on the base of [Azure official guide](https://docs.microsoft.com/en-us/azure/mysql/quickstart-create-mysql-server-database-using-azure-portal).
+
+### Configure Scalar DL 
+
+To apply the Scalar DL schema to MySQL, update the following configuration in [schema-loading-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/schema-loading-custom-values.yaml)
+
+```yaml
+database: jdbc
+contactPoints: <JDBC connection url>
+username: <MySQL username>
+password: <MySQL password>
+```
+
+To deploy Scalar DL on MySQL, update the following configuration in [scalardl-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/scalardl-custom-values.yaml) 
+
+```yaml
+dbContactPoints: <JDBC connection url>
+dbUsername: <MySQL username>
+dbPassword: <MySQL password>
+dbStorage: jdbc
+```
+
+### Scale resources
+
+After you create your server, you can independently change the vCores, the hardware generation, the pricing tier (except to and from Basic), the amount of storage, and the backup retention period. 
+The number of vCores can be scaled up or down. The storage size can only be increased. Scaling of the resources can be done either through the portal or Azure CLI.
+
+#### Storage auto-grow
+
+Storage auto-grow prevents your server from running out of storage and becoming read-only. If storage auto grow is enabled, the storage automatically grows without impacting the workload. 
+For servers with less than equal to 100 GB provisioned storage, the provisioned storage size is increased by 5 GB when the free storage is below 10% of the provisioned storage.
+For servers with more than 100 GB of provisioned storage, the provisioned storage size is increased by 5% when the free storage space is below the greater of 10 GB or 5% of the provisioned storage size. 
+
+### Backup and Restore
+
+Azure Database for MySQL automatically creates server backups and stores them in user configured locally redundant or geo-redundant storage. Backups can be used to restore your server to a point-in-time. 
```

Comment:
@brfrn169 Can you check if it is correct?

Reply:
If the Backup in Azure Database for MySQL is a consistent backup, it is correct.

`mysqldump` with `--single-transaction` can take a consistent backup:
https://www.percona.com/blog/2020/05/27/best-practices-for-mysql-backups/

 We can ask Azure Support if the Backup in Azure Database for MySQL is a consistent backup similar to `mysqldump`.

Reply:
Let me check again and update